src/ApiPort/ApiPort.Vsix/source.extension.vsixmanifest

```diff
@@ -11,11 +11,11 @@
         <Tags>code analysis, cross-platform, .NET Framework, .NET, portable, cross platform, Portable Library</Tags>
     </Metadata>
     <Installation>
-        <InstallationTarget Version="[15.0,16.0)" Id="Microsoft.VisualStudio.Pro" />
-        <InstallationTarget Version="[15.0,16.0)" Id="Microsoft.VisualStudio.Premium" />
-        <InstallationTarget Version="[15.0,16.0)" Id="Microsoft.VisualStudio.Ultimate" />
-        <InstallationTarget Version="[15.0,16.0)" Id="Microsoft.VisualStudio.Enterprise" />
-        <InstallationTarget Version="[15.0,16.0)" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Pro" />
+        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Premium" />
+        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Ultimate" />
+        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Enterprise" />
+        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Community" />
     </Installation>
```

Comment:
`[15.0,)` would work? I saw from [one of Mads' commit](https://github.com/madskristensen/FileIcons/commit/ff29c06b9b7fd3a405bc21d6a5601a39ff0d9787#diff-ae3ceb3342a0e975f3035fe2fcd91d13R15)